generators/ruby/faraday/templates/README.mustache

```diff
@@ -54,52 +59,46 @@ All 3 error classes provide the `response_code`, `response_body`, `json_body`, `
 
 ```ruby
 def create_applicant
-  onfido.applicant.create(params)
+  {{{gemName}}}.applicant.create(params)
 rescue Onfido::RequestError => e
   e.type          # => 'validation_error'
   e.fields        # => { "email": { "messages": ["invalid format"] } }
   e.response_code # => '422'
 end
 ```
 
-## Other configuration
+### Webhook event verification
 
-Optional configuration options with their defaults:
+Webhook events payload needs to be verified before it can be accessed. Library allows to easily decode the payload and verify its signature before returning it as an object for user convenience:
 
 ```ruby
-onfido = Onfido::API.new(
-  # ...
-  open_timeout: 10,
-  read_timeout: 30
-)
-```
+  require 'onfido/webhook_event_verifier'
+  verifier = Onfido::WebhookEventVerifier.new("_ABC123abc...3ABC123_")
 
-## Verifying webhooks
+  signature = "a0...760e"
 
-Each webhook endpoint has a secret token, generated automatically and [exposed](https://documentation.onfido.com/#register-webhook) in the API. When sending a request, Onfido includes a signature computed using the request body and this token in the `X-SHA2-Signature` header.
-
-You should compare this provided signature to one you generate yourself with the token to verify that a webhook is a genuine request from Onfido.
-
-```ruby
-if Onfido::Webhook.valid?(request.raw_post,
-                          request.headers["X-SHA2-Signature"],
-                          ENV['ONFIDO_WEBHOOK_TOKEN'])
-  process_webhook
-else
-  render status: 498, text: "498 Token expired/invalid"
-end
+  event = verifier.read_payload('{"payload":{"r...3"}}', signature)
 ```
```

Comment:
Let's include also here the try-catch block with exception thrown when webhook signature is invalid, so user immediately see how to use this feature.

Reply:
added